gdeploy.spec

```diff
@@ -3,7 +3,7 @@
 %global gdeploytemp %{_datadir}/gdeploy
 
 Name:           gdeploy
-Version:        2.0.8
+Version:        2.0.9
```

Comment:
There is no matchin tag in the deploy github repository yet?

```
$ spectool -g -S gdeploy.spec 
Getting https://github.com/gluster/gdeploy/archive/v2.0.9.tar.gz#/gdeploy-2.0.9.tar.gz to ./gdeploy-2.0.9.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   123    0   123    0     0    266      0 --:--:-- --:--:-- --:--:--   266
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found
```

Reply:
@nixpanic I have fixed the tags in origin, thanks for spotting it.